tests/compile-and-dump/Singletons/BadBoundedDeriving.ghc82.template

```diff
@@ -1,3 +1,6 @@
 
 Singletons/BadBoundedDeriving.hs:0:0: error:
     Can't derive Bounded instance for Foo_0 a_1.
+  |
+6 | $(singletons [d|
+  |   ^^^^^^^^^^^^^^...
```

Comment:
Do you think we should include these error message locations in golden files? They do add some degree of bloat.